app/frontend/styles/components/schools_in_your_trust.scss

```diff
@@ -0,0 +1,17 @@
+.missing-information {
+  border-top: 5px solid $govuk-brand-colour;
+  padding-top: govuk-spacing(2);
+}
+
+.schools {
+  border-top: 5px solid govuk-colour("orange");
```

Comment:
We should use single quotes here, I think

Reply:
This is something we need to talk about all together. I was of your same opinion of having single quotes and double quotes depending on whether we have interpolation or not. Lately DHH is suggesting for using double quotes everywhere to avoid noise in the review of PRs:

at around minute 54 it starts the topic of formatter:
https://railsconf.com/2020/video/david-heinemeier-hansson-keynote-interview-with-david-heinemeier-hansson